packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -77,7 +77,7 @@ import { StatefulChatClient } from '@internal/chat-stateful-client';
 import { ChatThreadClient } from '@azure/communication-chat';
 import { useEffect, useRef, useState } from 'react';
 import { _toCommunicationIdentifier } from '@internal/acs-ui-common';
-/* @conditional-compile-remove(unsupported-browser) */
+/* @conditional-compile-remove(rooms) */
```

Comment:
is the conditional compile statement for rooms didn't exist already for the import and if no, why do we need it now?

Reply:
after some previous refactoring only rooms uses this import, it should have had both `unsupported-browser` and `rooms` so this change should just remove the `unsupported-browser` but there was a gap where rooms wasn't specified (because we never test our beta compilation compiling only a subset of all features enabled)